config/config.test.js

```diff
@@ -0,0 +1,6 @@
+const config = {
+    mail: "office@wearesho.com",
+    tel: "380 66 024-94-02",
```

Comment:
Here should be just number. If you need another format in components - you should format in these components.